SpriterDotNet/Spriter.cs

```diff
@@ -266,17 +211,13 @@ public abstract class SpriterKey : SpriterElement
 
         [XmlAttribute("c4")]
         public float C4;
-
-        public SpriterKey()
-        {
-            Time = 0;
-            CurveType = SpriterCurveType.Linear;
-        }
     }
 
     [Serializable]
     public enum SpriterObjectType
     {
+        None = 0,
```

Comment:
I prefer not explicitly assigning values to an enum except when those numbers actually mean something (like HttpStatusCode). I find it similar to explicitly assigning false to a bool member variable, etc. Enums default to the first value and that's a know language feature so it should be fine as it is.